packages/frontend-v2/utils/theme/index.ts

```diff
@@ -48,17 +48,11 @@ const colors = {
     },
   },
   neutral: {
-    main: "#e7ebf1",
-    50: "#eff2f6",
-    100: "#d2d7de",
-    200: "#b3bcc8",
-    300: "#f1f5fa",
-    400: "#7586a0",
-    500: "#e7ebf1",
-    600: "#485569",
-    700: "#e2e6ec",
-    800: "#1f242c",
-    900: "#d2d8e2",
+    gray0: "#f4f6f9",
```

Comment:
Can you just change these to `gray.0`. So like gray would be its own object. This way we aren't doing `gray0`, `gray1`. It would instead be `gray.50`, or `gray.100`

Reply:
Hhhh gotcha i see what you mean now

Reply:
Would it be fine to just do `neutral.100` instead of `neutral.gray.100` since neutral is just gonna be for grays?

Reply:
Yea if we feel like there wouldn't be anymore neutrals, then this works